pkg/cmd/stack/translate.go

```diff
@@ -680,7 +684,13 @@ func translateResources(svc *model.Service) apiv1.ResourceRequirements {
 	return result
 }
 
-func getSvcProbe(svc *model.Service) *apiv1.Probe {
+type healthcheckProbes struct {
+	readiness *apiv1.Probe
+	liveness  *apiv1.Probe
+}
+
+func getSvcHealthProbe(svc *model.Service) *healthcheckProbes {
```

Comment:
If this function cannot retrieve a nil, I wouldn't return a pointer. There are a potential nil pointer in all places were the result of this function is used. Nothing prevents that in a future change we return a nil within this function and then, we fail in a non-controlled way